MSAL/test/unit/native_auth/controllers/MSALNativeAuthCredentialsControllerTests.swift

```diff
@@ -121,13 +121,14 @@ final class MSALNativeAuthCredentialsControllerTests: MSALNativeAuthTestCase {
 
         let expectedContext = MSALNativeAuthRequestContext(correlationId: defaultUUID)
```

Comment:
Added, but not here, but rather in `MSALNativeAuthResultFactoryTests` as that's where it should be. Also changed how the code works because the `MSALAccount` always gets created no matter what as it's not an optional (we had it wrong because of Objective C/Swift interoperability).
Changed the names of the tests on this class to highlight the `userAccountResult` doesn't get created rather than the `account`